css/templates/e-commerce/page.css

```diff
@@ -701,6 +747,9 @@ amp-selector [option][selected] {
   border-bottom: 1px dashed #dadada;
 }
 
+.amp-mode-mouse .commerce-listing-product:hover > .commerce-listing-product-name {
```

Comment:
i don't think hover works on mobile devices, so you can drop the amp-mode-mouse.  (same above)